src/esw/brushless.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python3
+import threading
+import time as t
+import asyncio
+import rospy
+import math
+from sensor_msgs.msg import JointState
+from geometry_msgs.msg import Twist
+from std_msgs.msg import Header
+import numpy as np
+import moteus
+
+
+# import moteus_pi3hat
+
+
+class CommandData:
+    MAX_TORQUE = 0.5
+    VELOCITY_LIMIT = 5  # TODO - Change after regenerative braking is solved
+
+    def __init__(self, position: float = math.nan, velocity: float = 0.0, torque: float = MAX_TORQUE):
+        self.position = position
+        self.velocity = max(-CommandData.VELOCITY_LIMIT, min(CommandData.VELOCITY_LIMIT, velocity))
+        self.torque = max(0, min(CommandData.MAX_TORQUE, torque))
+
+
+class MoteusBridge:
```

Comment:
In general why are there no comments / descriptions for the classes and major items? I have some in my branch and I think it is useful to maintain good descriptions. You can copy the ones from my version or can you write a quick blurb about classes and items like MoteusBridge, DriveApp, Application, etc.

Reply:
Idk comments are kinda bloated and I'm a believer of self documenting code... but ill add a few for the classes alone

Reply:
I think a sentence for the 3 major classes here is alright and not too bloated. There exists a happy medium in my opinion.